src/colp/components/usetype_version_comparison_report.py

```diff
@@ -0,0 +1,76 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import plotly.express as px
+from st_aggrid import AgGrid
+from src.constants import COLOR_SCHEME
+from abc import ABC
+import pdb
+
+
+class UsetypeVersionComparisonReport:
+    def __init__(self, usetype_changes, version):
+        self.usetype_changes = usetype_changes
+        self.version = version
+
+    def __call__(self):
+        st.subheader("Usetype Version Comparison")
+
+        if self.usetype_changes is None:
+            st.info("There is no usetype version comparison report for this branch.")
+            return
+
+        self.current_usetype_changes = self.filter_by_version(
+            self.usetype_changes, self.version
+        )
+
+        st.markdown(
+            f"""
+            This report compares the usetype distribution between the previous and selected versions.
+            Because there are a high number of usetypes, we have included a slider to be able to more clearly view a subsection.
+
+            **Current Version:** {self.version}
+
```

Comment:
The dates for the previous version and the current version aren't in the same format, I am assuming this is a product of how the digital ocean files were named/set in a colp build?

Reply:
yes that's correct unfortunately, outside the scope of this PR